t_bpf/test_bpf_base.py

```diff
@@ -0,0 +1,79 @@
+"""Bpf tests to check error handlings."""
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2024 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+from framework import tester
+from framework.parameterize import param, parameterize
+from framework.x509 import CertGenerator
+from helpers import dmesg, remote
+
+
+class TestBpfFailFunction(tester.TempestaTest):
+    backends = [
+        {
+            "id": "deproxy",
+            "type": "deproxy",
+            "port": "8000",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        }
+    ]
+
+    clients = [
+        {
+            "id": "deproxy",
+            "type": "deproxy",
+            "addr": "${tempesta_ip}",
+            "port": "80",
+        },
+    ]
+
+    tempesta = {
+        "config": """
+            listen 80;
+            server ${server_ip}:8000;
+        """
+    }
+
+    @parameterize.expand(
+        [
+            param(name="tfw_cli_conn_alloc", msg="can't allocate a new client connection"),
+            param(name="tfw_client_obtain", msg="can't obtain a client for frang accounting"),
+        ]
+    )
+    def test(self, name, msg):
+        self.start_all_services(client=False)
+
+        cmd = f"echo {name} > /sys/kernel/debug/fail_function/inject"
+        out = remote.client.run_cmd(cmd)
+        cmd = f"printf %#x 0 > /sys/kernel/debug/fail_function/{name}/retval"
+        out = remote.client.run_cmd(cmd)
+
+        cmd = "echo 100 > /sys/kernel/debug/fail_function/probability"
+        out = remote.client.run_cmd(cmd)
+        cmd = "echo 0 > /sys/kernel/debug/fail_function/interval"
+        out = remote.client.run_cmd(cmd)
+        cmd = "printf %#x -1 > /sys/kernel/debug/fail_function/times"
+        out = remote.client.run_cmd(cmd)
+
+        client = self.get_client("deproxy")
+        request = client.create_request(method="GET", headers=[])
+        client.start()
+
+        self.oops_ignore = ["ERROR"]
+        try:
+            client.send_request(request, "200")
+        except:
+            pass
```

Comment:
why is `try/except` needed here? Please add a comment about it or add type of an exception.